src/latex_formats.jl

```diff
@@ -8,7 +8,7 @@ Generate a PrettyTables LaTeX formatter for signed integers.
 function safe_latex_Signed_col(col::Integer)
   return (s, i, j) -> begin
                         j !== col && return s
-                        return safe_latex_Signed(s)
+                        return ismissing(s) ? " "^10 : safe_latex_Signed(s) 
```

Comment:
Why on 10 characters?

Reply:
For alignment, here: https://github.com/JuliaSmoothOptimizers/SolverBenchmark.jl/pull/64/files#diff-d7feddf2c2c4d7c9fe5790ee590485d7R127-R129

```
\( 1.00\)e\(+00\) &            &   & missing \\
                  & \(     1\) & a & missing \\
\( 3.00\)e\(+00\) & \(     3\) & b & notmiss \\\hline
```

Reply:
Right but we don't know what formatter has already been applied to the rest of the column, do we? What if the user formatted the first two columns on 12 characters?

Reply:
In the LaTeX case though, if the user passes a format it would bypass the `safe` function, right?
Alternatively, I can remove the sizes and use a single space, like in the `String` case. It would look like this:
```
\( 1.00\)e\(+00\) &   &   & missing \\
  & \(     1\) & a & missing \\
\( 3.00\)e\(+00\) & \(     3\) & b & notmiss \\\hline
```

Reply:
Yes it would bypass the `safe` function. So at least, your solution aligns well in the default case. Let's keep it. Thanks.